src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.csproj

```diff
@@ -1,239 +1,29 @@
-    <Compile Include="ApiUsage\DependencyInjection\Domain\DependencyRegistrationExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\DependencyRegistry.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\FullyGenericRegisterExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\NonGenericRegisterExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\RegisterDynamicObjectFactoryExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\RegisterExtensionPointExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\RegisterInstantiatedObjectExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\RegisterParentAwareFactoryExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\RegisterPluginForExtensionPointExpression.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\Domain\RegisterSubInterfaceExpression.cs" />
-    <Compile Include="Diagnostics.cs" />
-    <Compile Include="Immutability\AddImmutableExceptionsCodeFix.cs" />
-    <Compile Include="Immutability\ImmutableHelpers.cs" />
-    <Compile Include="Immutability\KnownImmutableTypes.cs" />
-    <Compile Include="Immutability\MutabilityInspectionResult.cs" />
-    <Compile Include="Immutability\MutabilityInspectionResultFormatter.cs" />
-    <Compile Include="Immutability\MutabilityInspector.cs" />
-    <Compile Include="ApiUsage\NotNullAnalyzer.cs" />
-    <Compile Include="ApiUsage\DependencyInjection\DependencyRegistrationsAnalyzer.cs" />
-    <Compile Include="Extensions\Microsoft.CodeAnalysis.cs" />
-    <Compile Include="Extensions\Microsoft.CodeAnalysis.CSharp.cs" />
-    <Compile Include="Immutability\ImmutabilityAnalyzer.cs" />
-    <Compile Include="ApiUsage\LaunchDarkly\FeatureDefinitionAnalyzer.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="ApiUsage\RpcAnalyzer.cs" />
-    <Compile Include="ApiUsage\ServiceLocator\OldAndBrokenServiceLocatorAnalyzer.cs" />
-    <Compile Include="ApiUsage\ServiceLocator\SingletonLocatorAnalyzer.cs" />
-    <Compile Include="Immutability\UnsafeStaticsAnalyzer.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="packages.config" />
-    <None Include="tools\install.ps1">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="tools\uninstall.ps1">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <Analyzer Include="..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.Analyzers.dll" />
-    <Analyzer Include="..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.CSharp.Analyzers.dll" />
+    <None Update="tools\*.ps1" CopyToOutputDirectory="Always" Pack="true" PackagePath="tools" />

```

Comment:
@mthjones

Reply:
Ahhh, my bad. Github thought this diff was too large to render, so Ctrl-F failed me. :(